prototype_source/quantization_in_pytorch_2_0_export_tutorial.rst

```diff
@@ -0,0 +1,630 @@
+(prototype) Quantization in PyTorch 2.0 Export Tutorial
+==============================================================
+
+Today we have `FX Graph Mode
+Quantization <https://pytorch.org/docs/stable/quantization.html#prototype-fx-graph-mode-quantization>`__
+which uses ``symbolic_trace`` to capture the model into a graph, and then
+perform quantization transformations on top of the captured model. In a
+similar way, for Quantization 2.0 flow, we will now use the PT2 Export
+workflow to capture the model into a graph, and perform quantization
+transformations on top of the ATen dialect graph. This approach is expected to
+have significantly higher model coverage, better programmability, and
+a simplified UX.
+
+Prerequisites:
+-----------------------
+
+-  `Understanding of the quantization concepts in PyTorch <https://pytorch.org/docs/master/quantization.html#quantization-api-summary>`__
+-  `Understanding of FX graph mode post training static quantization <https://pytorch.org/tutorials/prototype/fx_graph_mode_ptq_static.html>`__
+-  `Understanding of torchdynamo concepts in PyTorch <https://pytorch.org/docs/stable/dynamo/index.html>`__
+
+Imagine a backend developer who wishes to integrate a third-party backend
+with PyTorch's quantization 2.0 flow. To accomplish this, they would only need
+to define the backend specific quantizer. The high level architecture of
```

Comment:
I think we would need to mention what quantizer is here

Reply:
Any suggestions about how to introduce quantizer API of `get_supported_operators`, as we discussed here https://github.com/pytorch/tutorials/pull/2354#discussion_r1213834401. It's not finalized yet, but it's a must have when define a `Quantizer`.

Reply:
I think you can skip explaining that here, please focus on the role of Quantizer and the annotate function, you can also try to have a quick summarize of https://docs.google.com/document/d/1_jjXrdaPbkmy7Fzmo35-r1GnNKL7anYoAnqozjyY-XI/edit#heading=h.jtqauapwj95c here when talking about the motivation of the quantizer API

Reply:
Thanks. Add the introduction of ``Quantizer`` such as why we introduce this class and its APIs.